### ISSUE TITLE
<enter> doesn’t close the findbar during the first search, but then does

### ISSUE THREAD

**Issue by reaganf2** [akhodakivskiy/VimFx#531]
I just installed VimFx and am loving it! Just wanted to thank you first for writing this awesome add-on.

My only issue with it is related to the search behavior. In vim, after I hit </> and enter the search key, I usually hit <Enter> to go to the first search result, after which I hit <n/N> to navigate the search results.

In the case of VimFx, hitting <Enter> after typing out the search key unselects the first found instance of the search key on the webpage, and hitting <n> after that reselects this same instance again. It would be great if hitting <Enter> after typing out the search key just closes the search box while continuing to highlight the first search result, so that hitting <n> after that takes you to the second result, and so on.

FYI, I'm using VimFx version 0.5.17.1 on Firefox version 40.0.3 in Linux Mint.

**Comment by lydell** [comment]
Your desired behaviour is actually how VimFx works. You have run into #249. When you press `/` Firefox’s built-in `/` command is invoked (which opens the “quick find bar”) instead of invoking VimFx’s search command. This is fixed in the development version. For the time being, try changing VimFx’s `/` shortcut to something else.

**Comment by reaganf2** [comment]
I just changed the shortcut for "Enter find mode", and using the new shortcut opens the same search bar that the </> button initially opened. This is also the same search bar that opens when I choose "Find" from the Firefox menu. Now with the shortcut changed, hitting the </> key opens Firefox's quick find bar, which is different.

I also noticed that I have the issue I described above only when I first search for a key in a tab. When I search for the same keyword again, hitting <Enter> closes the search bar and leaves the first search result highlighted, as expected.
